src/modules/notifications.jsx

```diff
@@ -10,7 +10,7 @@ export function showNotification(err) {
     const errDetail = _.get(err.response, 'data.errorDetail');
     status = String(err.response.status);
     if (_.isEmpty(errDetail)) {
-      message = _.get(err.response, 'data.message');
+      message = _.get(err.response, 'data.message') ?? _.get(err.response, 'data.error.message');
```

Comment:
This allows us to get a more meaningful error message when a user logs in with a non-enterprise workspace google account (Which is not allowed to use this feature - https://docs.google.com/document/d/1YG7UpBpExxHViwcC-MIYji7mwhlh9vQSfYwxoSxIcpA/edit#heading=h.h4uq7f5zs05h)